chess/engine.py

```diff
@@ -2904,6 +2959,12 @@ def analysis(self, board: chess.Board, limit: Optional[Limit] = None, *, multipv
             future = asyncio.run_coroutine_threadsafe(coro, self.protocol.loop)
         return SimpleAnalysisResult(self, future.result())
 
+    def send_game_result(self, board: chess.Board, winner: Optional[Color] = None, game_ending: Optional[str] = None, game_complete: bool = True) -> None:
```

Comment:
Let's make sure `game_ending` cannot be used to inject arbitary commands (`}\n...`).

Options [are safe](https://github.com/niklasf/python-chess/blob/58a54f135f0f1a6d9272e307fde7053f316a7b17/chess/engine.py#L288), but a similar thing slipped through wrt. `Opponent`.

Reply:
Would getting rid of the newlines and curly braces be sufficient?
```python
game_ending = ' '.join(game_ending.split()).replace('{', '').replace('}', '')
```
Or, should an error be raised if the string contains new lines or curly braces?

Reply:
I went with raising an exception.

Reply:
> Let's make sure `game_ending` cannot be used to inject arbitary commands (`}\n...`).
> 
> Options [are safe](https://github.com/niklasf/python-chess/blob/58a54f135f0f1a6d9272e307fde7053f316a7b17/chess/engine.py#L288), but a similar thing slipped through wrt. `Opponent`.

Does `send_opponent_information()` need fixing? The UCI version goes through `configure()`, so that should be safe. What about the XBoard version?